src/AvenueClothing/Models/LoginModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace AvenueClothing.Models
+{
+    public class LoginViewModel
+    {
+        public string Login { get; set; }
+
+        public string Password { get; set; }
+
```

Comment:
I don't see a reason for the empty lines.

Reply:
Removed the empty lines.